examples/allennlp/allennlp_jsonnet.py

```diff
@@ -7,17 +7,10 @@
 Since it is too time-consuming to use the training dataset,
 we here use the validation dataset instead.
 
-We have the following two ways to execute this example:
+The example can be executed as follows:
 
-(1) Execute this code directly.
     $ python allennlp_jsonnet.py
```

Comment:
How about deleting this part (incl. the `The example can be executed as follows:` above) as well? At least for the ones that don't have pruning or other options. It doesn't really contain any useful information.

Reply:
Thanks @hvy for the quick review. You are right about the line not adding any useful information, therefore I have removed it from most places (except pruning) and have also updated the catalyst example.

Let me know if it looks good!

Reply:
Thanks a lot for the swift fix. It basically LGTM but I'm afraid there was a failure with the CI and I left some really small nits.